src/1-foundations/1-type-theory/exercises/1.2-solution.rzk.md

```diff
@@ -83,16 +91,27 @@ Projections from the recursion:
 
 ## Solution for dependant pairs
 
-$\Sigma$-types are built-in. Also, `pr₁-Σ`, `pr₂-Σ`, `rec-Σ`, `ind-Σ` were already defined in [chapter 1.6](../06-dependent-pair-types.rzk.md).
+Similarly, for $\Sigma$-types, the recursion principle has the type:
+
+$$ \mathsf{rec}_{\Sigma_{x:A}B(x)} : \Pi_{C:U}(\Pi_{x:A} B(x) \to C) \to (\Sigma_{x:A} B(x)) \to C $$
+
+and can be defined using the projection functions <!--($pr_1 :(\Sigma_{x:A} B(x)) \to A$ and $pr_2 : \Pi_{p:\Sigma_{x:A}B(x)}B(pr_1(p))$)--> as such:
```

Comment:
Why commented out text? Should be removed?

Reply:
Oh, yes, that's a mistake. I was thinking whether I should write these types or if it would be too verbose but then forgot to delete them